prow/integ-suite-kind.sh

```diff
@@ -171,6 +171,16 @@ fi
 if [[ -z "${SKIP_BUILD:-}" ]]; then
   trace "setup kind registry" setup_kind_registry
   trace "build images" build_images "${PARAMS[*]}"
+  WASM_ATTRGEN_TAG="359dcd3a19f109c50e97517fe6b1e2676e870c4d"
+  docker pull gcr.io/istio-testing/wasm/attributegen:$WASM_ATTRGEN_TAG
+  docker tag gcr.io/istio-testing/wasm/attributegen:$WASM_ATTRGEN_TAG localhost:5000/istio-testing/wasm/attributegen:$WASM_ATTRGEN_TAG
+  docker push localhost:5000/istio-testing/wasm/attributegen:$WASM_ATTRGEN_TAG
+  if [[ "$IP_FAMILY" == "ipv6" ]]; then
```

Comment:
Can you expand on why we cannot resolve over IPv6? It works on my machine, but maybe I have a weird setup?

Reply:
Resolving `kind-registry` does not work for me on IPv6-only cluster (I use the latest docker engine). I was getting the following error in istiod:
```
2024-06-28T10:02:45.008513Z     warn    delta   ADS:ECDS: ACK ERROR b-v1-9c6d95958-7jwqr.echo-3-15056-11 Internal:cannot fetch Wasm module oci://[fd00:10:96::b04f]:1338/istio-testing/wasm/attributegen:359dcd3a19f109c50e97517fe6b1e2676e870c4d: could not fetch Wasm OCI image: could not fetch manifest: Get "https://kind-registry:5000/v2/istio-testing/wasm/attributegen/manifests/359dcd3a19f109c50e97517fe6b1e2676e870c4d": dial tcp: lookup kind-registry on [fd00:10:96::a]:53: server misbehaving
```
And I've seen in kube-dns logs that it was trying to connect to 172.18.0.1 (default address in /etc/resolv.conf), but was getting "connection refused". I found similar issues: https://github.com/kubernetes-sigs/kind/issues/3114, https://github.com/moby/moby/issues/41651, and it looks like it's still not supported.